.travis.yml

```diff
@@ -39,14 +39,28 @@ jobs:
 
 deploy:
   - provider: script
-    script: bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
+    script: >-
```

Comment:
Also earlier in the Travis file I see:

- stage: Run black formatter and commit any changes
      if: branch != master
      script:
        - bash ci_tools/new_commit_with_black.sh

If this triggers, what happens to the rest of the Travis job? Does it test the now out of date code or does it stop given that it has pushed a commit and invalidated the code it is running against?

Reply:
A new commit triggers a new build which cancels the old build.
This is configured by the auto-cancellation functionality of Travis:
https://blog.travis-ci.com/2017-03-22-introducing-auto-cancellation